src/EditorFeatures/CSharpTest/AddParameter/AddParameterTests.cs

```diff
@@ -2407,5 +2407,76 @@ public class C {
 }";
             await TestMissingAsync(code);
         }
+
+        [WorkItem(29061, "https://github.com/dotnet/roslyn/issues/29061")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddParameter)]
+        public async Task TestThis_DontOfferToFixTheConstructorWithTheDiagnosticOnIt()
+        {
+            // error CS1729: 'C' does not contain a constructor that takes 1 arguments
+            var code =
+@"
+public class C {
+    
+    public C(): [|this|](1)
```

Comment:
this one doesn't offer. but is there existing one that support create constrcutor?

Reply:
The create constructor/method is a separate fixer. I haven't check whether that one is offered here. But in case it isn't it I would file a new issue.